app/models/host.rb

```diff
@@ -306,10 +306,8 @@ def ipmi_power_reset
   end
 
   def reset
-    if supports?(:reset)
+    if verbose_supports?(:reset)
```

Comment:
note: the message coming out of here changed from "can not stop" to "can not reset"

I'm assuming this is right, but had to ask